src/elektra/redshift.ini

```diff
@@ -0,0 +1,148 @@
+; Elektra spec file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+; Based on: https://github.com/ElektraInitiative/libelektra/blob/master/examples/spec/lcdproc/LCDd.ini
+
+; TODO:
+; 6. add command line arguments to specification and split up colon-separated options for location-providers and adjustment-methods into separate CLI arguments.
+
+[]
+mountpoint = redshift.ecf
+infos/plugins = ni validation
+
+; Configuration values - section redshift
+;=======================================================
+[temp/day]
+type = unsigned_short
+description = The color temperature the screen should have during daytime.
+default = 6500
+
+[temp/night]
+type = unsigned_short
+description = The color temperature the screen should have during nighttime.
+default = 4500
+
+[fade]
+type = boolean
+description = Whether transitions between color temperatures (e.g. from daytime to nighttime) should happen fast or slow.
+default = 1
+
+[brightness]
+type = float
+description = The screen brightness.
+default = 1.0
+check/range = 0-1
+
+[brightness/day]
+type = float
+description = The screen brightness during daytime.
+default = 1.0
+check/range = 0-1
+
+[brightness/night]
+type = float
+description = The screen brightness during nighttime.
+default = 1.0
+check/range = 0-1
+
+[elevation/high]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds).
+default = 3.0
+
+[elevation/low]
+type = float
+description = By default, Redshift will use the current elevation of the sun to determine whether it is daytime, night or in transition (dawn/dusk). When the sun is above the degrees specified with the elevation-high key it is considered daytime and below the elevation-low key it is considered night (source: https://github.com/jonls/redshift/wiki/Configuration-file#solar-elevation-thresholds).
+default = -6.0
+
+[gamma]
+type = float
+description = The screen gamma.
+default = 1.0
+
+[gamma/day]
+type = float
+description = The screen gamma during daytime.
+default = 1.0
+
+[gamma/night]
+type = float
+description = The screen gamma during nighttime.
+default = 1.0
+
+[adjustment-method]
+type = string
```

Comment:
What is the default? There should be always either default or require (only if default is impossible to provide).

Reply:
The default is OS-specific (e.g. macOS -> quartz, Windows -> w32gdi, Linux -> rest). Is there a way to handle that in Elektra?
I would leave it required and have a default config file provided in the package. That's how it is done currently.

Reply:
You can consider Elektra's spec language as a tool box, which can be extended as wanted. But in this case it is probably not needed, the default seems to be quite static. So probably we should simply use the [pre-processor (cpp)](https://www.man7.org/linux/man-pages/man1/cpp.1.html) also for the spec.

For now simply add the 3 defaults as comments in the spec (maybe with surrounding `#ifdef _WIN32` without actually using a preprocessor, just to indicate what we plan to do).

Reply:
Thanks, done.

Reply:
Currently in Redshift, If no value is specified I believe it tries all available methods until it finds one that works. You could potentially have a default called "default", "any" or something like that, that selects this approach?

Reply:
Great idea, I've incorporated that.